src/main/java/org/cascadebot/cascadebot/commands/music/PlayingCommand.java

```diff
@@ -0,0 +1,64 @@
+package org.cascadebot.cascadebot.commands.music;
+
+import com.sedmelluq.discord.lavaplayer.track.AudioTrack;
+import lavalink.client.player.IPlayer;
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Member;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.messaging.MessagingObjects;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.utils.FormatUtils;
+
+import java.util.Set;
+
+public class PlayingCommand implements ICommandMain {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        IPlayer player = context.getData().getMusicPlayer().getPlayer();
+        AudioTrack track = player.getPlayingTrack();
+
+        if (player.getPlayingTrack() == null) {
+            context.getTypedMessaging().replyWarning("No music playing!");
+        } else {
+            EmbedBuilder embedBuilder = MessagingObjects.getClearThreadLocalEmbedBuilder();
+            embedBuilder.setTitle(track.getInfo().title + " - " + track.getInfo().author, track.getInfo().uri);
+            embedBuilder.setThumbnail("https://img.youtube.com/vi/" + track.getIdentifier() + "/hqdefault.jpg");
+            embedBuilder.addField("Status", player.isPaused() ? "\u23F8 Paused" : "\u25B6 Playing", true);
+            embedBuilder.addField("Duration", track.getInfo().isStream ? "This is a livestream!" : FormatUtils.formatLongTimeMills(track.getDuration()), true);
+            embedBuilder.addField("Amount played", FormatUtils.formatLongTimeMills(track.getPosition()), true);
```

Comment:
You should use the progress bar method i have instead of these two, and then add another field thats amount played and make it be `amount/duration` also you got the amount payed wrong you need to get the amount played dirrectly from the player not from the track.

Reply:
The get position works fine according to testing?

Reply:
With lavaplayer yes as, lavalink no. I ran into this on flare.